Jenkinsfile

```diff
@@ -106,9 +113,22 @@ spec:
                         crane push image.tar $IMAGE_NAME:$TAG3
                     elif [ $BRANCH_NAME == "master" ]; then
                         crane push image.tar $IMAGE_NAME:$TAG3
-                        git tag $VERSION
-                        git push origin --tags
                         crane push image.tar $IMAGE_NAME:$TAG4
+                        if [ $(git tag -l "$VERSION") ]; then
+                            echo "ERROR: Tag with version $VERSION already exists! Exiting."
+                        else
+                            # Recover some things we've lost:
+                            git config --global user.email "helx-dev@lists"
```

Comment:
is `git` going to work here? I know you added a container named `git` above but this stage is using the `crane` container, so I'm not sure how this will work

Reply:
Yes, I tested it, and it works. I originally added a git container but it wasn't needed because git as a tool is installed in Jenkins itself.

Reply:
does that mean we can remove the git container added above? if so I think we should

Reply:
Good catch. Removed it.